scripts/contributor_dashboard_debug.py

```diff
@@ -0,0 +1,262 @@
+            json={
+                'email': email,
+                'password': password
+        }).json()['idToken']
+
+        return token_id
+
+    def _get_csrf_token(self) -> str:
+        """Gets the CSRF token."""
+        response = self._make_request('GET', '/csrfhandler')
+        csrf_token = str(
+            json.loads(response.text[len(feconf.XSSI_PREFIX):])['token'])
+
+        return csrf_token
+
+    def _assign_admin_roles(self, roles: List[str], username: str) -> None:
+        """Assigns the given roles to the user with the given username."""
+        for role in roles:
+            params = {
+                'payload': json.dumps({
+                    'role': role,
+                    'username': username
+                }),
+                'csrf_token': self.csrf_token
+            }
+            self._make_request(
+                'PUT', feconf.ADMIN_ROLE_HANDLER_URL, params=params)
+
+    def _add_submit_question_rights(self, username: str) -> None:
+        """Adds submit question rights to the user with the given username."""
+        params = {
+            'payload': json.dumps({'username': username}),
+            'csrf_token': self.csrf_token
+        }
+
+        self._make_request(
+            'POST', '/contributionrightshandler/submit_question', params=params)
+
+    def _generate_sample_new_structures_data(self) -> None:
+        """Generates sample new structures data."""
+        params = {
+            'payload': json.dumps({
+                'action': 'generate_dummy_new_structures_data'}),
+            'csrf_token': self.csrf_token
+        }
+
+        self._make_request('POST', '/adminhandler', params=params)
+
+    def _add_topics_to_classroom(
+            self, classroom_name: str, classroom_url_fragment: str) -> None:
+        """Adds all dummy topics to a classroom."""
+        response = self._make_request(
+            'GET', '/topics_and_skills_dashboard/data')
+        topic_summary_dicts = json.loads(
+            response.text[len(feconf.XSSI_PREFIX):])['topic_summary_dicts']
+        topic_ids = [
+            topic_summary_dict['id'] for topic_summary_dict in
+            topic_summary_dicts]
+
+        params = {
+            'payload': json.dumps({
+                'action': 'save_config_properties',
+                'new_config_property_values': {
+                    'classroom_pages_data': [{
+                        'name': classroom_name,
+                        'url_fragment': classroom_url_fragment,
+                        'course_details': '',
+                        'topic_list_intro': '',
+                        'topic_ids': topic_ids
+                    }]
+                }
+            }),
+            'csrf_token': self.csrf_token
+        }
+
+        self._make_request('POST', '/adminhandler', params=params)
+
+    def _make_request(
+            self,
+            method: str,
+            url: str,
+            params: Dict[str, Any] | None = None,
```

Comment:
You'll need to add a comment above on why `Any` is used for `params` above. 

The comment should be of the format
`# Here we use type Any because [reason]`

Ditto elsewhere `Any` is used

Reply:
I replaced all `Any` with specific types.